R/taxonomy.R

```diff
@@ -134,7 +134,7 @@ NULL
 #' @format a \code{character} vector of length 8 containing the taxonomy ranks
 #'   recognized. In functions this is used as case insensitive.
 #' @export
-TAXONOMY_RANKS <- c("domain","kingdom","phylum","class","order","family",
+TAXONOMY_RANKS <- c("domain","superkingdom","kingdom","phylum","class","order","family",
```

Comment:
I'm not sure if we want to add superkingdom here since domain and superkingdom mean same thing